sympy/physics/mechanics/body.py

```diff
@@ -0,0 +1,165 @@
+from sympy import Symbol, symbols
+from sympy.physics.mechanics import (RigidBody, Particle, ReferenceFrame,
+                                     inertia)
+from sympy.physics.vector import Point, Vector
+
+__all__ = ['Body']
+
+
+class Body(RigidBody, Particle):
+    """
+    Body is a common representation of RigidBody or a Particle.
+
+    A Body represents either a rigid body or particle in classical mechanics.
+    Bodies have a body-fixed reference frame, a mass, a mass center and
+    possibly a body-fixed inertia.
+
+    Parameters
+    ----------
```

Comment:
I typically explain the parameter in both places. For example:

```
masscenter : Point, optional
    A point that represents the mass center of the body or particle. If no point is given a point is generated.
```

Something like that.